src/game/CAvaraApp.cpp

```diff
@@ -460,6 +460,7 @@ bool CAvaraAppImpl::DoCommand(int theCommand) {
 
 OSErr CAvaraAppImpl::LoadLevel(std::string set, std::string levelTag, CPlayerManager *sendingPlayer) {
     SDL_Log("LOADING LEVEL %s FROM %s\n", levelTag.c_str(), set.c_str());
+    if (itsGame->itsFilm->HasUnflushedFrames()) itsAPI->RecordFrames(itsGame);
```

Comment:
Need a way to enable/disable saving game recordings, like a check against `kSaveGameRecordings`, off by default?